packaging/pkg_helpers.bash

```diff
@@ -157,7 +157,9 @@ setup_wheel_python() {
     conda create ${CONDA_CHANNEL_FLAGS} -yn "env$PYTHON_VERSION" python="$PYTHON_VERSION"
     conda activate "env$PYTHON_VERSION"
     # Install libpng from Anaconda (defaults)
-    conda install ${CONDA_CHANNEL_FLAGS} libpng "jpeg<=9b" -y
```

Comment:
Is this script used? The `jpeg<=9b` is definitely stale as we no longer pin the library.